en/services/gimbal_v2.md

```diff
@@ -0,0 +1,356 @@
+
+#### Overriding
+
+Overriding means that anything set is ignored.
+It is generally discouraged to use this unless something doesn’t work as intended, a mission is set-up incorrectly, etc.
+
+
+### Custom gimbal device settings
+
+Custom gimbal settings can be accomplished using the component information microservice which is based on a [component definition file](../services/component_def.md) similar to the [camera definition file](../services/camera_def.md).
+
+## FAQ
+
+### How to set System ID?
+
+The system ID of all components (e.g. autopilot, companion computer, camera, gimbal) on a drone/system need to be the same.
+This needs to be either done manually by configuration, or alternatively, the components need to listen to the heartbeat of the autopilot and then adjust their system ID accordingly.
+
+### When is gimbal device also gimbal manager?
+
+The default case should be to use the gimbal manager in the autopilot.
+The only exception to this are integrated solutions containing a camera and gimbal for functionality like like visual tracking.
+
+### How to test gimbal device?
+
+A gimbal device can be tested by connecting it to an autopilot with a gimbal manager implemented and active.
+To avoid having to do a full setup including autopilot, a direct test using MAVSDK is planned.
+
+TODO: add link to MAVSDK test once it exists.
+
+### How to control a gimbal with the old MAVLink protocol?
+
+Gimbals that use the (old) Gimbal protocol v1 should still be supported by autopilot software.
+Basically, the gimbal manager needs to translate the commands to the old protocol.
+
+### How to control a gimbal without MAVLink support?
+
+Gimbals controlled using a protocol like PPM, PWM, SBUS or something proprietary can still be supported.
+The autopilot will have to act as the gimbal manager and provide the driver and translation to the respective protocol.
+
+### What about RC (non-MAVLink) control?
+
+The autopilot needs to be configured to either accept MAVLink input (so [GIMBAL_MANAGER_SET_ATTITUDE](#GIMBAL_MANAGER_SET_ATTITUDE)) or RC control.
+
+For RC control, the channels will have to be manually mapped/configured to control the gimbal.
+It is up to the gimbal manager implementation to deconflict between RC and MAVLink input.
+This is in the same way that also RC input to fly needs to be selected from either RC or MAVLink and is up to the implementation.
+The recommendation is to make it configurable using for instance a parameter.
+
+
+## Message/Command/Enum Summary
+
+### Gimbal Manager Messages
+
+This is the set of messages/enums for communicating with the gimbal manager (by GCS, autopilot, etc.).
+
+Message | Description
+-- | --
+<span id="MAV_CMD_DO_GIMBAL_MANAGER_ATTITUDE"></span>[MAV_CMD_DO_GIMBAL_MANAGER_ATTITUDE](../messages/common.md#MAV_CMD_DO_GIMBAL_MANAGER_ATTITUDE) | High level setpoint to be sent to a gimbal manager to set a gimbal attitude. Note: a gimbal is never to react to this command but only the gimbal manager.
```

Comment:
This is a command. @hamishwillee is there a reason that this appears under "messages"?

Reply:
I've addressed it in #277b10f, let me know if you agree.

Reply:
Nope, I'm all good with this. Often I don't separate the two things in these tables, but it is probably good to do so 

Thank you.